### PR TITLE
[KOGITO-8179]-Fix Operator imports with Container-builder

### PR DESCRIPTION
**Pull request by desmax74**
See: https://issues.redhat.com/browse/KOGITO-8179
Signed-off-by: desmax74 <mdessi@redhat.com>

### CODE REVIEW

**Comment by desmax74** [comment]
@spolti @davidesalerno

**Review by davidesalerno — APPROVED** [review]

File: builder/testdata/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,5 @@
-FROM quay.io/kiegroup/kogito-serverless-workflow-base:latest AS builder
+#Not yet released Change the 31 st oct with quay.io/kiegroup/kogito-serverless-workflow-base:latest
+FROM quay.io/mdessi/kogito-serverless-workflow-base:latest AS builder
```
Comment (spolti) [inline builder/testdata/Dockerfile]: don't we have it in the kiegroup repo  yet?
if not, can you please push it?

Comment (desmax74) [reply builder/testdata/Dockerfile]: @spolti We have a nightly version https://quay.io/repository/kiegroup/kogito-swf-builder-nightly only @radtriste can push

Comment (spolti) [reply builder/testdata/Dockerfile]: can we use this one?

Comment (spolti) [reply builder/testdata/Dockerfile]: or push a snahpshot?
If we can use a snapshot, i can push it.

Comment (desmax74) [reply builder/testdata/Dockerfile]: Fixed with "quay.io/repository/kiegroup/kogito-swf-builder-nightly" until 31 oct with the official version

**Review by spolti — APPROVED** [review]

**Pull request merged**